lib/Net/Google/Drive/Simple.pm

```diff
@@ -262,6 +262,45 @@ sub file_upload {
     return $file_id;
 }
 
+###########################################
+sub rename {
+###########################################
+    my ( $self, $file_id, $new_name ) = @_;
+
```

Comment:
maybe want to check that `$file_id` and `$new_name` are provided by the caller

Reply:
Am I missing something, or do none of the other functions do that? E.g. `sub file_create` and `sub file_upload`?

I'm not saying you're wrong to desire it - I just copied the conventions of the existing code, or thought I did, and would rather add error checking to every sub as a separate patch if you want.

Reply:
I understand that you mimic the other functions. Was just making general comments to the provided code. The same comments are valid for the other functions too.
Overtime we are all improving the way we code, we should not hide behind bad habits.

Reply:
a separate patch is fine if you want